examples/java/spring-boot/src/main/java/uk/gov/api/springboot/infrastructure/models/daos/ApiDao.java

```diff
@@ -1,6 +1,6 @@
-package uk.gov.api.springboot.daos;
+package uk.gov.api.springboot.infrastructure.models.daos;
 
-public class MetadataDao {
+public class ApiDao {
```

Comment:
It may be worth converting this to a `record` in the meantime, so we:

- don't have to write unnecessary tests for SonarQube
- have (maybe) an easier type to play with before we add the persistence layer